TPMainController.m

```diff
@@ -40,7 +40,7 @@
 extern CGSError	CPSGetFrontProcess( CPSProcessSerNum *pPSN);
 extern CGSError CPSSetFrontProcess( const CPSProcessSerNum *pPSN);
 
-#define VERSION_CHECK_URL @"http://www.abyssoft.com/software/teleport/version.plist"
+#define VERSION_CHECK_URL @""
```

Comment:
Should this also be set to `https://invalid.invalid` for the time being?

Reply:
Nothing uses this define at the moment since it's commented out, but I've made the change anyway since it will be easier to find both locations once sparkle updates are re-enabled.